test-common/src/main/java/com/mentoring/api/RequestInterface.java

```diff
@@ -0,0 +1,9 @@
+package com.mentoring.api;
+
+import io.restassured.http.Cookies;
+
+import io.restassured.response.Response;
+
+public interface RequestInterface {
```

Comment:
It doesn't necessary to include the Interface word into the name of the class since it's an interface.

Reply:
Actually this is a leftover from previous experiments, and is not needed. It's now removed.